roles/openshift_health_checker/test/sdn_tests.py

```diff
@@ -170,6 +171,33 @@ def execute_module(module_name, args, *_):
     SDNCheck(execute_module, task_vars).run()
 
 
+def test_resolve_address():
+    def execute_module(module_name, args, *_):
+        if module_name != 'command':
+            raise ValueError('not expecting module %s' % module_name)
+
+        command_args = args['_raw_params'].split()
+        if command_args[0] != '/bin/getent':
+            raise ValueError('not expecting command: %s' % args.raw_params)
+
+        # The expected command_args is ['/bin/getent', 'ahosts4', 'foo'].
```

Comment:
it is in a comment .. but typo `ahosts4`  aka should be ```['/bin/getent', 'ahostsv4', 'foo']```

Reply:
Thanks! Fixed.